Cargo.toml

```diff
@@ -5,11 +5,12 @@ members = [
     "light-client",
     "light-client-js",
     "p2p",
+    "pbt-gen",
     "proto",
     "rpc",
     "tendermint",
-    "testgen",
-    "pbt-gen"
+    "test",
```

Comment:
> Is the idea here to have a dedicated crate for cross-crate integration testing?

That's one of the use-cases for this crate.

> It might be good to add a `README.md` to the `test` crate to explain what it's for.

Good shout.